deps/src/ideals.cpp

```diff
@@ -404,7 +393,6 @@ void singular_define_ideals(jlcxx::Module & Singular)
 
   Singular.method("id_res", &id_res_helper);
   Singular.method("id_mres_map", &id_mres_map_helper);
-  Singular.method("id_prune_map", &id_prune_map_helper);
```

Comment:
You remove `id_prune_map` but there is a call to it in `src/module/module.jl` line 405

Reply:
Specifically in the implementation of the function `prune_with_map`, which also has a test case.

Reply:
re-added id_prune_map_helper